app/src/main/java/org/greenstand/android/TreeTracker/models/Locations.kt

```diff
@@ -271,17 +276,6 @@ class Convergence(val locations: List<Location>) {
     var latitudeConvergence: ConvergenceStats? = null
         private set
 
-    private fun computeStats(data: List<Double>): ConvergenceStats {
```

Comment:
Minor change. Moved the `private fun` to the bottom of the class. Prefer to have public functions at the beginning of the class and then privates.